scripts/generate-conda-envs.py

```diff
@@ -109,6 +109,7 @@ def conda(self) -> Reqs:
             "make",
             "rust",
             "ninja",
+            "pkg-config",
```

Comment:
I suggest we explicitly add `openssl` here, even though it's pulled by other packages.

Reply:
https://github.com/nv-legate/legate.core/pull/778/commits/a1e81d4aa1beda94255cea4f78c67814dc39bf90